py-polars/polars/utils/convert.py

```diff
@@ -82,16 +93,16 @@ def _timedelta_to_pl_duration(td: timedelta | str | None) -> str | None:
 
 def _datetime_to_pl_timestamp(dt: datetime, time_unit: TimeUnit | None) -> int:
     """Convert a python datetime to a timestamp in nanoseconds."""
-    dt = dt.replace(tzinfo=timezone.utc)
+    dt = dt.replace(tzinfo=timezone.utc) if dt.tzinfo != timezone.utc else dt
```

Comment:
is this one safe to do?

because otherwise, won't `datetime` use your system's time zone? e.g. for me, in the UK, I see:
```python
In [5]: datetime(2023, 3, 26, 2).timestamp()
Out[5]: 1679792400.0

In [6]: datetime(2023, 3, 26, 1).timestamp()
Out[6]: 1679792400.0
```

slightly concerned that the test suite didn't catch this, will look into it

Reply:
I think the semantic is the same as `dt = dt.replace(tzinfo=timezone.utc)` since it just skips the `replace` operation when it's already in utc timezone?

Reply:
aah sorry, I'd misread

all good then, thanks!